src/component/Footer/Footer.js

```diff
@@ -41,17 +52,25 @@ class Footer extends Component {
       event.target.reset();
     } else if (inputUserEmail) {
       alert("현재 지원되지 않는 기능입니다. 조금만 기다려주세요! ⏰");
-    } else {
-      return;
     }
   };
 
   render() {
     const { inputEmail } = this.state;
     const { handleInput, onKeyDown, subScribeByEmail } = this;
+    const USER_VALIDATION = {
+      isValidatedEmail: () =>
+        inputEmail.includes("@") && inputEmail.length >= 8,
+    };
```

Comment:
시도 너무 좋습니다!

다만 이렇게 하면 isValidateEmail를 한 번만 사용할 수 있기 때문에 함수로 분리한 다음 인자로 넣어서 return 값을 true false로 나올 수 있게 바꿔 주세요!

Reply:
리뷰 감사합니다 관희님. 그런데 현재 이 validation 코드는 해당 컴포넌트에서 단 한 번만 사용되는 코드입니다. 따라서 재사용 될 일이 없는 코드입니다. 이런 상황에서는 위와 같은 형태로 사용하는 것이 괜찮지 않을까 하는 생각이었는데, 혹시 관희님께서는 어떻게 생각하실까요?? :) 

재사용 여부와 관계 없이 분리해주는 것이 좋은 것인가요?!

Reply:
이 부분에 대해서 관희님의 의견을 다시 한 번만 여쭙고 싶어서 우선 바로 리뷰 요청 라벨을 달아두려 합니다! :)

Reply:
저는 그렇다면 차라리 우선 함수로 먼저 만들고 추후에 다른 곳에서 사용하게 되면 utils로 분리할 것 같습니다!

```suggestion
const checkValidEmail = (email) => {
  return email.includes("@) && email.length >=8
}
```